listenbrainz_spark/stats/sitewide/artist.py

```diff
@@ -1,49 +1,36 @@
 from listenbrainz_spark.stats import run_query
-from pyspark.sql.functions import collect_list, sort_array, struct
 
 
-def get_artists(table: str, date_format: str):
+def get_artists(table: str, limit: int = 1000):
```

Comment:
Should 1000 be a constant or documented somewhere?

Reply:
This is the default the code was using till now. I was thinking to make this configurable through spark request stats commands. Do you think that's needed or just making this a constant is fine?

Reply:
I would guess a constant is fine for now -- until we realize that we do really need it to be changeable.